hatch/app.py

```diff
@@ -119,7 +124,12 @@ def workspace_release(
         raise HTTPException(400, errors)
 
     response = models.create_release(workspace, workspace_dir, release, token.user)
-    # forward job-servers response back to client
+    release_id = response.headers["Release-Id"]
+
+    # rewrite location header to point to our upload endpoint
```

Comment:
Can you add a note here around why we're doing this?  I think something along the lines of:

    … so clients will upload files via release-hatch